.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
```

Comment:
### These are suggestions for future projects:

1. In future projects please ensure you have a meaningful branch name that conveys an action or a feature name that you trying to implement, for example: `setup-hello-microvers` or `hello-microverse` ...etc
2. Also ensure that the PR has a descriptive Title and summary that explains your achievements or the changes you made for the milestone, as an example:
    - **Title:** In this PR I made the following changes:
    - **Summery:**
        - Create the `.html` file.
        - Setup linters for html-css project.
        - ...etc
3.  Also ensure your repo includes meaningful commit messages, a good habit for a professional developer is committing often for each change is made 👍🏼